client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -752,31 +753,36 @@ export function YourBrowserSupportToolkit() {
       <h4>Vendor prefixes</h4>
       <p>
         The browser compat tables on MDN will also tell you if the property or
-        value you want to use needs to be prefixed. Vendor prefixes are a way
-        for browsers to ship their own experimental CSS features that won’t
-        interfere with the standard version of the property, as well as
+        value you want to use needs to be <em>prefixed</em>. Vendor prefixes are
+        a way for browsers to ship their own experimental CSS features that
+        won't interfere with the standard version of the property, as well as
         browser-specific proprietary features, using a prepended prefix.
       </p>
       <p>Prefixes used in major browsers are as follows:</p>
       <ul>
         <li>
-          -webkit- (Chrome, Safari, newer versions of Opera, Edge after version
-          79, almost all iOS browsers including Firefox for iOS, any other
-          Chromium-based browsers not already mentioned — including Brave,
-          Samsung Internet, etc.)
+          <code>-webkit-</code> (Chrome, Safari, newer versions of Opera, Edge
+          after version 79, almost all iOS browsers including Firefox for iOS,
+          any other Chromium-based browsers not already mentioned &mdash;
+          including Brave, Samsung Internet, etc.)
+        </li>
+        <li>
+          <code>-moz-</code> (Firefox)
```

Comment:
The code background color doesn't read well against the background, could you change the code background to light grey 30 (#E0E0E6) at 50%?